### ISSUE TITLE
Add a "zap gremlins" feature to automate gremlins removal

### ISSUE THREAD

**Issue by nhoizey** [nhoizey/vscode-gremlins#23]
> On a side note but related would be nice if there was a change all option when the gremlins are highlighted. BBEdit has a feature called "Zap Gremlins" that does this. Its basically just a regex find and replace with a dialog. May be out of scope for this project but would love that ability. I have had smart quotes/curly quotes cost me hours trying to debug errors from those things popping up in code. 
> 
> There is an extension that I am going to use for this purpose just wish it was on github so I could contribute :( [Replace Smart Characters](https://marketplace.visualstudio.com/items?itemName=DrMattSm.replace-smart-characters#review-details)

_Originally posted by @disaac in https://github.com/nhoizey/vscode-gremlins/issues/8#issuecomment-369776535_

**Comment by ctf0** [comment]
i was going to request the exact same thing

**Comment by ctf0** [comment]
@nhoizey check https://github.com/ctf0/vscode-replace-pesky-characters

**Comment by nhoizey** [comment]
@ctf0 thanks for the link, I'll see how I can add this feature to Gremlins.

**Comment by TheSench** [comment]
I've started looking at this one.  I'll have a PR up sometime in the next week that will at least have zapping current document.

**Comment by TheSench** [comment]
PR is ready, just waiting on the language-overridable PR as it builds upon the configuration changes made there.

**Comment by TheSench** [comment]
Branch still ready, just waiting on #160, as I built the PR off of that branch.  I can undo those changes if necessary, but just need confirmation.
